apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -102,11 +102,12 @@ type MicroservicesStatus struct {
 // ModeType is the authentication/authorization mode in which Tempo Gateway
 // will be configured.
 //
-// +kubebuilder:validation:Enum=dynamic
+// +kubebuilder:validation:Enum=static
```

Comment:
it should list all enum values

Reply:
dynamic is not implemented. thats why i removed it from the list.